src/components/RoomView/setupGame.jsx

```diff
@@ -9,13 +9,17 @@ function SetupGame() {
   const [selectedWordPackId, setSelectedWordPackId] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
   const roomId = useSelector((state) => state.player.roomId);
-  //   //----------------fet all packs for users to select from-----------------//
+
+  const dispatch = useDispatch();
```

Comment:
useDispatch is not imported, dispatch seems to not be in use in this file so perhaps you could just remove line 13!